pandevice/objects.py

```diff
@@ -702,3 +702,73 @@ def _setup(self):
             condition={'action_type': 'tagging'})
 
         self._params = tuple(params)
+
+
+class ScheduleObject(VersionedPanObject):
+    """Schedule Object
+
+    Args:
+        name (str): Name of the object
+        disable_override (bool): True to set disable-override
+        type (str): Type of Schedule: recurring or non-recurring
+        recurrence (str): daily or weekly recurrence
+        non_recurring_date_time (list/str): date and time range string for a non-recurring schedule
+                Example: 2019/11/01@00:15-2019/11/28@00:30
+        daily_time (list/str): time range for a daily recurring schedule
+                Example: 17:00-19:00
```

Comment:
Same with this; you can differentiate between the two by using "short time range" and "long time range."